src/pipecat/services/aws.py

```diff
@@ -146,7 +148,9 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
             # Filter out None values
             filtered_params = {k: v for k, v in params.items() if v is not None}
 
-            response = self._polly_client.synthesize_speech(**filtered_params)
+            response = await asyncio.to_thread(
+                self._polly_client.synthesize_speech(**filtered_params)
```

Comment:
I think this needs this update otherwise it's treated as callable.

```suggestion
                lambda: self._polly_client.synthesize_speech(**filtered_params)
```

Reply:
Oops, sorry. It has to be (updated):

```
            response = await asyncio.to_thread(
                self._polly_client.synthesize_speech, **filtered_params
            )
```